src/controllers/simplified-account/settings/worldpay-details/flex-credentials/worldpay-flex-credentials.controller.test.js

```diff
@@ -62,6 +62,10 @@ describe('Controller: settings/worldpay-details/flex-credentials', () => {
 
     it('should pass context data to the response method', () => {
       mockResponse.should.have.been.calledWith(sinon.match.any, sinon.match.any, sinon.match.any, {
+        credentials: {
+          organisationalUnitId: undefined,
```

Comment:
🤔, I think you should set a value on the request and check that it appears rather than testing for undefined